vscode/vscode/vscode.ml

```diff
@@ -2521,6 +2652,271 @@ module TreeView = struct
   end
 end
 
+module WebviewPanelOptions = struct
+  include Interface.Make ()
+
+  include
+    [%js:
+    val enableFindWidget : t -> bool [@@js.get]
```

Comment:
Possible undefined. Should return `bool or_undefined`.

https://github.com/microsoft/vscode/blob/1.58.2/src/vs/vscode.d.ts#L7529

Reply:
Done in af072b6330ac64e439013b2f099033607d85d9f5